### PR TITLE
Show case bug fixes and feature additions

### PR DESCRIPTION
**Pull request by VillanuevaK**

### CODE REVIEW

File: src/main/java/codeu/controller/FileUploadServlet.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 
 /* This servlet will load upon startup and map it to the URL /FileUploadServlet. */
-@WebServlet("profile/FileUploadServlet")
+@WebServlet({"profile/FileUploadServlet", "profileView/FileUploadServlet"})
```
Comment (yufangqq) [inline src/main/java/codeu/controller/FileUploadServlet.java]: Some comments about the difference between these two URLs?

File: src/main/java/codeu/controller/ProfileViewServlet.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import java.util.UUID;
+import java.util.HashMap;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+/* Servlet class responsible for the activity feed page. */
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ProfileViewServlet.java]: Update comment?

Comment (yufangqq) [reply src/main/java/codeu/controller/ProfileViewServlet.java]: Comment on how is this different from ProfileServlet?

File: src/main/java/codeu/model/data/Event.java [hunk 3]
```diff
@@ -183,9 +183,12 @@ public Event(User user) {
     userId = user.getId();
 	}
 
+  public String getNameOfUser() {
```
Comment (yufangqq) [inline src/main/java/codeu/model/data/Event.java:186]: Indent/align with the rest of the code in this file.

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 4]
```diff
@@ -23,15 +23,14 @@ List<Event> events = (List<Event>) request.getAttribute("events");
   }
 </style>
 
-
-<script>
+    <script>
     // scroll the feed div to the bottom
-      function scrollFeed() {
+      function feedChat() {
```
Comment (yufangqq) [inline src/main/webapp/WEB-INF/view/activity.jsp]: Why is this renamed?
Based on comment "// scroll the feed div to the bottom" seems the original scrollFeed() might be a better name.

File: src/main/webapp/WEB-INF/view/profileView.jsp [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  #feed {
+    background-color: white;
+    height: 500px;
+    overflow-y: scroll
+  }
+</style>
+
+<script>
+    // scroll the feed div to the bottom
+      function scrollFeed() {
```
Comment (yufangqq) [inline src/main/webapp/WEB-INF/view/profileView.jsp:31]: This is not renamed to feedChat()?
Better to be consistent with the naming if the same method is defined in different places.

A even better approach is to define this scrollFeed() only ONCE in some JS file, and include that in the JSP files where it is needed.

**Review by yufangqq — APPROVED** [review]

File: src/main/java/codeu/controller/FileUploadServlet.java (hunk 1 above)
Comment (VillanuevaK) [reply src/main/java/codeu/controller/FileUploadServlet.java]: Done.

File: src/main/java/codeu/controller/ProfileViewServlet.java (hunk 2 above)
Comment (VillanuevaK) [reply src/main/java/codeu/controller/ProfileViewServlet.java]: Updated the comment.

File: src/main/java/codeu/model/data/Event.java (hunk 3 above)
Comment (VillanuevaK) [reply src/main/java/codeu/model/data/Event.java:186]: Done

File: src/main/webapp/WEB-INF/view/activity.jsp (hunk 4 above)
Comment (VillanuevaK) [reply src/main/webapp/WEB-INF/view/activity.jsp]: I wasn't really thinking about the name, but how when I was calling the function I was calling it by a different name. This has been fixed.

File: src/main/webapp/WEB-INF/view/profileView.jsp (hunk 5 above)
Comment (VillanuevaK) [reply src/main/webapp/WEB-INF/view/profileView.jsp:31]: All of them should be scrollFeed() now. It is a fairly simple function, but you're right, since it is used so much it might make sense to make it streamlined in that sense.
